packages/components/src/DateTimePickers/pickers/DateTimePicker/DateTimePicker.component.js

```diff
@@ -187,7 +188,11 @@ class DateTimePicker extends React.Component {
 				aria-label="Date picker"
 			>
 				{viewElement}
-				<div className={theme.footer}>
+				<div
+					className={classNames(theme.footer, {
+						[theme['date-padding']]: this.state.isDateTimeView,
```

Comment:
Time picker within dateTimePicker will be deprecated soon. I don't think we should bother with that.
I suggest you to set the line within date picker sass file only to set the line above and under the calendar, so it's in the same file, managed the same way, and it will survive the current date picker refacto where we split date and time.
It's not a problem that the time part doesn't have the line, it will be removed.

Reply:
if  i set only in date picker sass, then there will be no underline in the month picker.
For now the DateTimePicker has a `switcher` between date picker and month picker, so it is best place for the underline.
And after the DateTimePicker deprecated, it is easy to move the footer(with the Today button) and the underline to the new `switcher`. I will work with Wang Yan that time for it.

Reply:
Ok let's reset: what bothers me the most is that `.date-padding`that add a padding top in one case and not in the other.
Can you move that to have this space managed by the date and the month part to have a margin-bottom specifically, instead of managing it at this level ?
Then you can remove that classname

Reply:
Not only you, the underline bothered me for long time since html table doesn't support margin things.
I moved the margin for the underline several times already. From viewlayout to footer at last.
For now the default margin-top is $padding-normal witch is designed for month picker, and the `.date-padding` will override it in date picker.